test/basic.ssr.test.js

```diff
@@ -24,7 +24,10 @@ test.serial('Init Nuxt.js', async t => {
       stats: false
     },
     render: {
-      csp: true
+      csp: {
+        hashAlgorithm: 'sha256',
+        allowDomains: ['https://example.com', 'https://example.io']
```

Comment:
Please set  `allowedDomains` in `common/options.js` with an empty array to indicate such options is available

Reply:
Thank you for review!
I changed to the contents for the review, is it okay so?

Reply:
Yes. Thanks :)